shared/common-adapters/choice-list.desktop.tsx

```diff
@@ -37,7 +38,10 @@ const rawCSS = `
     background-color: transparent;
   }
   .cl-entry:hover {
-    background-color: ${globalColors.blueLighter2};
```

Comment:
let's actually move this to a real css import instead of this inline css thing. Also i think we should hide the existence of darkColors from all code

Reply:
Done ✔️